### DISCUSSION TITLE
Port checker, ssl checker and dig

### DISCUSSION THREAD

**Discussion by MadalinNitu** [CorentinTh/it-tools#328] [Ideas]
Hey, very nice toolset. There are only three things missing and it would be perfect for my usecases:

- port checker .. like ping.eu telnet to check a ip/port if it's open
- ssl checker - giving a certificate to show information about it like expiration date, issuer etc or check remote with openssl connect
- dig dns for a record TXT, A, CNAME etc

Very good work, you've done!

**Comment by sharevb** [comment]
Hi @MadalinNitu, unfortunately, it-tools is a pure client side tool, so it cannot perform server side things like opening socket to test ssl, port or make DNS queries

**Comment by sharevb** [comment]
Hi @MadalinNitu, DNS query can be made over HTTPS, #1371, if you arr interested in up to date version of it-tools, I made kind of a fork here : https://github.com/sharevb/it-tools (https://sharevb-it-tools.vercel.app/ and docker images https://github.com/sharevb/it-tools/pkgs/container/it-tools)
